src/components/Login/Login.jsx

```diff
@@ -39,7 +39,7 @@ function Login(props) {
 					</b>`);
 			})
 			.catch(() => {
-				Alertify.error(`<b style='color:white;'>Email y/o password erroneos
+				Alertify.error(`<b style='color:white;'>Email y/o Password Erróneos
```

Comment:
Contraseña instead of Password please, only upper case the first E
Email y/o contraseña erróneos 
```suggestion
				Alertify.error(`<b style='color:white;'>Email y/o contraseña erróneos
```